boards/riscv/beagle_v_starlight/beagle_v_starlight_defconfig

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
as per @pdp7's recommendation, I would change to `beaglev_starlight_jh7100`

Reply:
Fixed.